packages/chaindata-provider/src/types/Chain.ts

```diff
@@ -39,6 +39,8 @@ export type Chain = {
   // TODO: Delete (has its own store now)
   /** @deprecated has its own store now */
   balancesMetadata: Array<BalancesMetadata>
+
+  hasCheckMetadataHash?: boolean // Indicates if the chain has the CheckMetadataHash extension, enabling signing with ledger generic app
```

Comment:
Just curious, how do we identify if a chain has this extension, and do we need to maintain this manually in chaindata?

Reply:
to know, need to download the metadata and see if there is a the CheckMetadataHash entry in the extrnsinsic's extension type

It's done here automatically in chaindata :
https://github.com/TalismanSociety/chaindata/blob/feat/hasCheckMetadataHash-flag/scripts/fetch-external/steps/updateChainsExtrasCache.ts#L361